### ISSUE TITLE
Generate public key with standard curve 'secp256r1'(NIST Curve P-256 )

### ISSUE THREAD

**Issue by hamedaakhlaghi** [agens-no/EllipticCurveKeyPair#34]
How can I generate a public key with specific domain parameters (a, b, p, x, y, n, h) according to secp256r1

**Comment by hfossli** [comment]
I have not done that before. Do you want to do that whilst storing the private key on secure enclave? I don't think that is possible.

You could create a key pair outside of the sdk and import the public key to the sdk. I can help you with importing.

**Comment by hamedaakhlaghi** [comment]
I saw `secp256r1` in the readme, I thought your library support this standard. I don't want to store public or private, I need to generate public key with secp256r1.

**Comment by hfossli** [comment]
Yes, it supports the same configuration for secp256r1 as Security framework from Apple provides of support. I don't think I can help you further than that. Maybe if you share some pseudo code?

**Comment by Sajjon** [comment]
@hamedaakhlaghi I [support `secp256r1` in EllipticCurveKit](https://github.com/Sajjon/EllipticCurveKit/blob/master/Source/EllipticCurve/CommonCurves/Secp256r1.swift)

**Comment by hamedaakhlaghi** [comment]
@Sajjon Thanks, My problem resolved

**Comment by hfossli** [comment]
How did you resolve?

**Comment by hamedaakhlaghi** [comment]
@hfossli  I used https://github.com/ricmoo/GMEllipticCurveCrypto framework.
